### DISCUSSION TITLE
HabitKit

### DISCUSSION THREAD

**Discussion by sebroehl** [ABausG/home_widget#289] [Show and tell]
Hey everyone,

just wanted to show off the widgets I built with home_widget for my habit tracking app [HabitKit](https://habitkit.app). I also utilize the HomeWidgetBackgroundWorker to support iOS Shortcuts and Automations. Thanks for making this awesome package, Anton!

![image](https://github.com/user-attachments/assets/d2e2aed4-f709-48f3-b583-726f04786f32)

**Comment by pavchhab** [comment]
Love the app! just wish you had any 2 by 1's on android, I can stretch it just fine and it adjusts but the blocks become a bit too big and it starts to show less history.
